packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -83,7 +83,7 @@ export default function ProfileMenu({
       {
         title: 'Logout',
         buttonProps: {
-          icon: <PowerIcon />,
+          icon: <ExitIcon />,
```

Comment:
This was [requested in Figma](https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/daily.dev---Dev-Mode?type=design&node-id=999-2766&mode=design&t=uW89qeqTHwNsyIK7-0)